ci/log.py

```diff
@@ -44,19 +44,29 @@ class Bcolors:
     BLUE = '\033[34m'
 
 
-def configure_default_logging(stdout_level=None):
+def configure_default_logging(
+    stdout_level=None,
+    force=False,
+):
     if not stdout_level:
         stdout_level = logging.INFO
 
     # make sure to have a clean root logger (in case setup is called multiple times)
-    logging.root.handlers.clear()
+    if force:
+        for h in logging.root.handlers[:]:
```

Comment:
why not simply use `handlers.clear()`? also, you can omit the slicing `[:]`
edit: ok, I suppose you want to not only rm the handlers, but also close them - correct?
still, no slicing operator required

Reply:
This is the way the python logging lib does it and the `logging.root.removeHandler()` method includes locking. I think you are right about the slicing tho it looks a bit redundant.

Reply:
Yes, its about closing them as well.
Sure, will remove the slicing.